src/v3/vite.config.ts

```diff
@@ -95,8 +95,8 @@ export default defineConfig(({ mode, command }) => {
       return {
         ...base,
 
-        // hide sourcemaps
-        sourcemap: false, // boolean | 'inline' | 'hidden'
+        // generate sourcemaps
+        sourcemap: true, // boolean | 'inline' | 'hidden'
```

Comment:
Is this change intentional?

Reply:
Yessir! The verify-package.js file expects the `okta-signin-widget.next.js` file to have a corresponding `okta-signin-widget.next.js.map` file for the sourcemaps, and if it does not exist it will fail the assertion. This setting generates that sourcemap file for us.